.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Question
+about: Ask a question about the ONNX.
+title: ''
+labels: 'question'
+assignees: ''
+
+ 
+
+---
+# Ask a Question
+
+### Question
+Explain your question here.
+
+### Further information
+- Relevant Area (*e.g. model usage, backend, best practices, converters, shape_inference, version_converter, training, test, operators*): 
```

Comment:
Let's removes converter from relevant area. We want people to use converter repos for any questions\bugs